src/Makefile

```diff
@@ -1132,7 +1132,7 @@ ifdef TEST
 	OSSEC_LDFLAGS+=${LDFLAGS_TEST}
 endif #TEST
 
-test_programs = test_os_zlib test_os_xml test_os_regex test_os_crypto test_os_net test_shared
+test_programs = test_os_zlib test_os_xml test_os_regex test_os_crypto test_shared
```

Comment:
Would this not be slightly better? 

TEST_CHILD_PROGRAMS+=test_os_zlib
TEST_CHILD_PROGRAMS+=test_os_xml
ifneq($TRAVIS, true)
TEST_CHILD_PROGRAMS+=test_os_net
endif
test_programs = ${TEST_CHILD_PROGRAMS}